app/test/helpers/repositories.ts

```diff
@@ -49,10 +51,10 @@ export function setupFixtureRepository(repositoryName: string): string {
     entry => Path.basename(entry.path) === '_git'
   )
 
-  submodules.forEach(entry => {
+  submodules.forEach(async entry => {
```

Comment:
This change looks like it's moving us from synchronous to asynchronous without awaiting. I don't know if these submodule directories needs to be renamed in sequence or not but I would expect `setupFixtureRepository` to guarantee that all operations are completed when its resultant promise resolves.

Either we make this a regular `for` statement and await each or (if we don't care about the internal order) we map submodules into promises and await all of them using `Promise.all`

Reply:
@niik good shout - I'll ensure this isn't fire-and-forget